config/config_test.go

```diff
@@ -0,0 +1,53 @@
+package config_test
+
+import (
+	"os"
+
+	. "github.com/tidepool-org/platform/config"
+	"github.com/tidepool-org/platform/store"
+
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+)
+
+var _ = Describe("Config", func() {
+
+	var _ = Describe("FromJson", func() {
+
+		It("should load the given config file", func() {
+			var mgoConfig store.MongoConfig
+			FromJSON(&mgoConfig, "mongo.json")
+			Expect(mgoConfig).To(Not(BeNil()))
+			Expect(mgoConfig.URL).To(Not(BeEmpty()))
+		})
+		It("should error if the config doen't exist", func() {
+			var random interface{}
+			err := FromJSON(&random, "random.json")
+			Expect(random).To(BeNil())
+			Expect(err).ToNot(BeNil())
+		})
+	})
+
+	var _ = Describe("FromEnv", func() {
```

Comment:
Do we need the `var _ = ` here for it to work?  (I don't know.)  If not, let's get rid of it.